asset_dashboard/static/js/planner.js

```diff
@@ -168,7 +205,7 @@ class PortfolioPlanner extends React.Component {
 
     const data = {
       name: this.state.portfolio.name,
-      user: props.user_id,
```

Comment:
props is coming from python and styled as snake case, unless this was styled as camelCase somewhere i'm missing?

Reply:
I changed the props to camel case in the view code.